MaterialGraphProject/Assets/UnityShaderEditor/Editor/Drawing/Controls/ColorControl.cs

```diff
@@ -12,15 +12,17 @@ namespace UnityEditor.ShaderGraph.Drawing.Controls
     public class ColorControlAttribute : Attribute, IControlAttribute
     {
         string m_Label;
+        bool m_Hdr;
 
-        public ColorControlAttribute(string label = null)
+        public ColorControlAttribute(string label = null, bool hdr = false)
```

Comment:
passing bools to functions like this seems like the simple way but passing an enum ends up being better. 
1) You can add more values to the ENUM if needed
2) It's clear from usages what you are passing, right now I have to browse to the constructor to see what true / false means.